app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::ProjectsController < ApplicationController
+  before_action :set_project, only: %i[show update destroy]
```

Comment:
Codes look fine but the format seems to be different from the other two controllers, should we follow either format for the presentation?

Reply:
Since the %i[] format is a little more concise I suggest we refactor the other components to match.